sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,13 +1,37 @@
 # Release History
 
-## 11.9.0-beta.1 (Unreleased)
+## 11.9.0-beta.1 (2025-10-23)
```

Comment:
I'd leave this as `Unreleased` for now so we're forced to update it when actually shipping.